### PR TITLE
Create linux-headers.buildinfo

### PR DESCRIPTION
**Pull request by mhendrickx**

### CODE REVIEW

File: cucumber/dev-base/linux-headers/linux-headers.buildinfo [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+# Cucumber Linux Buildinfo for linux-headers
+
+NAME=linux-headers
+VERSION=4.16.8
+URL=(https://cdn.kernel.org/pub/linux/kernel/v4.x/linux-4.16.8.tar.xz)
```
Comment (Z5T1) [inline cucumber/dev-base/linux-headers/linux-headers.buildinfo]: Please use the $VERSION variable whenever possible. This makes it easier when we have to upgrade the package.

File: cucumber/dev-base/linux-headers/linux-headers.buildinfo [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Copyright 2016 Scott Court
```
Comment (Z5T1) [inline cucumber/dev-base/linux-headers/linux-headers.buildinfo]: Please make sure to add your name and the current year here, i.e.
`# Copyright 2018 John Doe`

**Comment by Z5T1** [comment]
A couple of required files are missing: slack-desc must be present, and there must be some sort of integrity verification (see https://github.com/cucumberlinux/ports/blob/master/utilities/documentation/requirements-for-package-submission, number 4).

File: cucumber/dev-base/linux-headers/linux-headers.buildinfo [hunk 3]
```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Copyright 2018 Scott Court, Maarten Hendrickx
```
Comment (Z5T1) [inline cucumber/dev-base/linux-headers/linux-headers.buildinfo]: Usually I do this like:
``
# Copyright 2016 Scott Court
# Copyright 2018 Maarten Hendrickx
``

Comment (mhendrickx) [reply cucumber/dev-base/linux-headers/linux-headers.buildinfo]: I will try to do this from now

**Pull request merged**